src/Containers/VendorsList/TagContainer.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Tag, TagProps } from '../Tag';
+import { MainInterface, ResponsiveInterface } from '../../Utils/BaseStyles';
+import { flex, media } from '../../Utils/Mixins';
+
+export interface ITagContainerProps extends MainInterface, ResponsiveInterface, React.HTMLAttributes<HTMLDivElement> {
+    tags: string[];
+    isHoverable?: boolean;
+    tagProps?: Omit<TagProps, 'children'>;  
```

Comment:
I would explore why children was added as a prop in the first place btw, otherwise this is good